### PR TITLE
Re-generate Code from Schema 1.0.1

### PR DESCRIPTION
**Pull request by boyleconnor**
Addresses nlpsandbox/nlpsandbox-schemas#179

Depends on #31

### CODE REVIEW

**Comment by tschaffter** [comment]
@cascadianblue You can also update `docker-compose.yml` to use the version `1.0.1` of the dependencies (their `tool` version happens to be `1.0.1` for all of them).

File: docker-compose.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ services:
       - physical-address-annotator
 
   phi-deidentifier-app:
-    image: nlpsandbox/phi-deidentifier-app:0.3.2
+    image: nlpsandbox/phi-deidentifier-app:1.0.0
```
Comment (tschaffter) [inline docker-compose.yml:44]: The latest version of the PHI Deidentifier is 1.0.1

```suggestion
    image: nlpsandbox/phi-deidentifier-app:1.0.1
```

**Review by tschaffter — CHANGES_REQUESTED** [review]
One comment

File: docker-compose.yml (hunk 1 above)
Comment (cascadianblue) [reply docker-compose.yml]: @tschaffter the name of this branch, as per [your instructions](https://github.com/nlpsandbox/nlpsandbox-schemas/issues/179), is `release-1.0.0`. I don't think we ever released a branch `1.0.0`.

**Pull request merged**